### PR TITLE
S04_US29 : Not mandatory inputs specified

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

File: src/View/Admin/Home/Component/_form_service.html.twig [hunk 1]
```diff
@@ -11,13 +11,16 @@
...
         <input class="price30min" type="number" name="price30min" placeholder="35" value="{{ services.price30min }}">
     </div>
     <div>
-        <label for="description">Votre message :</label>
+        <label for="description">Votre message* :</label>
         <textarea name="description" id="description" rows="4" placeholder="Description du service" required>{{ services.description }}</textarea>
     </div>
+    <div class="not-mandatory">
+        <p>Les champs ornés d'un astérisque sont facultatifs*<p>
+    </div>
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_form_service.html.twig:23]: counter-intuitive : just put "(facultatif)" in place of the asterisks ;)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by khadimb — APPROVED** [review]
ok

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**